Sources/App/Github/GithubCommandRouter.swift

```diff
@@ -0,0 +1,47 @@
+import Vapor
+
+// Basic router implementation
+public class GithubCommandRouter: Service {
+    /// The internal router
+    private let router: TrieRouter<Responder>
```

Comment:
good idea using a router for this 👍

Reply:
Yeah, I might rewrite my own bot/command hanlers to use one now :P